### ISSUE TITLE
What's limiting the WaveNet's receptive field?

### ISSUE THREAD

**Issue by strongrex2001** [ibab/tensorflow-wavenet#303]
My GTX1080 can hold about 200000 samples of audio, more than 10 seconds, however, the receptive field is only about ~300ms, and when I tried simply increasing the dilation depth with the max dilation rate of 8192 to get a receptive field of ~4 seconds, it seemed that the model's loss was not going to go down ~3.1.
By the way, from a implemention in keras, I saw the Dense layer connected to the flatten layer contained most of the parameters (more than 90%), and I didn't think that it was a good situation...

Thank you for answering!
